dev_update_configs_playbook.yml

```diff
@@ -12,25 +12,21 @@
       - secret_group_vars/dev_secrets
   handlers:
     - name: galaxy gravity restart
-      become: True
-      become_user: galaxy
-      command: "{{ galaxy_venv_dir }}/bin/galaxyctl graceful"
-      environment:
-        GRAVITY_STATE_DIR: "{{ galaxy_gravity_state_dir }}"
-        GALAXY_CONFIG_FILE: "{{ galaxy_config_file }}" 
+      command: "{{ galaxy_venv_dir }}/bin/galaxyctl -c {{ galaxy_config_file }} graceful"
+      listen: "restart galaxy"
```

Comment:
The listen task is named differently elsewhere.
https://github.com/usegalaxy-au/infrastructure/blob/d193aadea3cfb7fa8acdfe354f4dac6d3f173230/group_vars/galaxyservers.yml#L25

Reply:
That variable is deprecated. https://github.com/galaxyproject/ansible-galaxy#optional-variables I'll remove it.

Reply:
I also saw several other places that have "Restart Galaxy" instead of "restart galaxy"

Reply:
I can't find a single one with a text search